wa/framework/target/descriptor.py

```diff
@@ -4,14 +4,14 @@
 from devlib import (LinuxTarget, AndroidTarget, LocalLinuxTarget,
                     Platform, Juno, TC2, Gem5SimulationPlatform,
                     AdbConnection, SshConnection, LocalConnection,
-                    Gem5Connection)
+                    Gem5Connection, ChromeOsTarget)
```

Comment:
Really knit-picky, but I would stick the new Target import just after the existing Target imports.